src/Microsoft.IdentityModel.Clients.ActiveDirectory.Platform/Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.csproj

```diff
@@ -41,7 +41,7 @@
   <PropertyGroup Condition="'$(TargetFramework)' == 'net45'">
 	<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>$(DefineConstants);DESKTOP</DefineConstants>
-    <NoWarn>$(NoWarn);CS8002</NoWarn>
```

Comment:
>>$(NoWarn);CS8002 [](start = 11, length = 17)

it is not obvious to me why these are not needed for net45, but they are for Android and iOS. Does the other platforms also need these rules to be disabled?

Reply:
Im not sure either, need to ask @jmprieur about this one. I also dont know why we suppress 1701;1702;1705 In the projects. any Idea @jmprieur?